client/analytics/report/customers/table.js

```diff
@@ -102,9 +103,8 @@ export default class CustomersReportTable extends Component {
 	}
 
 	getCountryName( code ) {
-		const countries = ( wcSettings.dataEndpoints && wcSettings.dataEndpoints.countries ) || [];
-		const country = countries.find( c => c.code === code );
-		return country ? country.name : null;
+		const country = typeof countries[ code ] !== 'undefined' ? countries[ code ] : null;
+		return country;
```

Comment:
We could skip the `country` assignment and directly return. I think it will not affect code readability.